src/app/logs/logs.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, OnInit } from '@angular/core';
+import { RestService } from '../rest/rest.service';
+import { Router } from '@angular/router';
+
+
+declare var $: any;
+declare var Chartist: any;
+
+
+@Component({
+    selector: 'logs-cmp',
+    templateUrl: './logs.component.html'
+})
+
+export class LogsComponent implements OnInit {
+
+
+    public LogFileText: any;
+
+    constructor(private restService:RestService, private router:Router) { }
+
+    ngOnInit() {
+    }
+    ngAfterViewInit() {
+        this.getConsoleLogFile();
+    }
+
+    ngOnDestroy() {
+    }
+
+    getConsoleLogFile (): void{
+
+        this.restService.getConsoleLogFile().subscribe(data => {
+
+            this.LogFileText = data;
+
+        });
+
+    }
+
+
+    logChanged(event:any){
+
+        if(event == "ConsoleLog") {
```

Comment:
Please do not use hard coded strings in statements. AFAIK, these definitions are also defined in Web UI